src/util/psbt/mod.rs

```diff
@@ -19,17 +19,22 @@
 //! except we define PSBTs containing non-standard sighash types as invalid.
 //!
 
-use core::cmp;
-
-use crate::blockdata::script::Script;
-use crate::blockdata::transaction::{ TxOut, Transaction};
-use crate::consensus::{encode, Encodable, Decodable};
-use crate::consensus::encode::MAX_VEC_SIZE;
-pub use crate::util::sighash::Prevouts;
+use core::{fmt, cmp};
+use core::ops::Deref;
 
 use crate::prelude::*;
-
 use crate::io;
+
+use crate::blockdata::script::Script;
+use crate::blockdata::transaction::{EcdsaSighashType, Transaction, TxOut};
+use crate::consensus::{Encodable, Decodable};
+use crate::consensus::encode::{self, MAX_VEC_SIZE};
+use crate::util::bip32::{ExtendedPubKey, KeySource};
+use crate::util::ecdsa::{EcdsaSig, EcdsaSigError};
+use crate::util::key::PrivateKey;
+use crate::util::sighash::{self, SighashCache};
+use crate::secp256k1::{Message, Secp256k1, Signing};
```

Comment:
Why the `crate::` prefix here? My understanding is that we can directly import this here?

Reply:
Yes, I agree. At one stage I was using `crate::secp256k1::foo` but now realize that from within `rust-bitcoin` plain `secp256k1::foo` is better. Will fix.